.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,47 @@
+name: Question
+description: Ask a question
+title: 'question: '
+labels: [question]
+
+body:
+- type: textarea
+  id: question
+  attributes:
+    label: Question
+  validations:
+    required: true
+- type: textarea
+  id: maintainers
+  attributes:
+    label: Maintainer CC
+    description: |
+      Please @ people who are in the `meta.maintainers` list of the module in question.
+      If in doubt, check `git blame` for whoever last touched something.
+  validations:
+    required: false
```

Comment:
```suggestion
    required: true
```

Reply:
I did apply the change. But it’s worth taking into account that some questions might relate to modules or internal library functions that do not have a maintainer. 

Having users enter a space might still be the better option though.